homeassistant/components/switch/__init__.py

```diff
@@ -30,6 +30,8 @@
 
 ATTR_TODAY_MWH = "today_mwh"
 ATTR_CURRENT_POWER_MWH = "current_power_mwh"
+ATTR_CURRENT_POWER_W = "Current power (W)"
+ATTR_TOTAL_CONSUMPTION_KWH = "Total energy (kWh)"
```

Comment:
Attribute names should not be formatted like this. They should be current_power_w and total_energy_kwh.